From e0db12a9b5716afa00080817a736ccd8541ce411 Mon Sep 17 00:00:00 2001
From: KT <karl.funlab@gmail.com>
Date: Fri, 16 Jan 2015 13:15:15 +0800
Subject: [PATCH] board: sama5d3_xplained:

Check-in config file default SDRAM CONFIG_RAM_256MB
The SDRAM memory size CONFIG_MEM_SIZE is auto selected by CONFIG_RMA_xMB
---
 board/sama5d3_xplained/sama5d3_xplainednf_linux_uimage_dt_defconfig | 3 +--
 board/sama5d3_xplained/sama5d3_xplainednf_linux_zimage_dt_defconfig | 3 +--
 board/sama5d3_xplained/sama5d3_xplainednf_uboot_defconfig           | 2 +-
 board/sama5d3_xplained/sama5d3_xplainedsd_linux_uimage_dt_defconfig | 3 +--
 board/sama5d3_xplained/sama5d3_xplainedsd_linux_zimage_dt_defconfig | 3 +--
 board/sama5d3_xplained/sama5d3_xplainedsd_uboot_defconfig           | 2 +-
 6 files changed, 6 insertions(+), 10 deletions(-)

diff --git a/board/sama5d3_xplained/sama5d3_xplainednf_linux_uimage_dt_defconfig b/board/sama5d3_xplained/sama5d3_xplainednf_linux_uimage_dt_defconfig
index f0658b2..7af3eef 100644
--- a/board/sama5d3_xplained/sama5d3_xplainednf_linux_uimage_dt_defconfig
+++ b/board/sama5d3_xplained/sama5d3_xplainednf_linux_uimage_dt_defconfig
@@ -1,10 +1,9 @@
 CONFIG_SAMA5D3_XPLAINED=y
 CONFIG_CPU_CLK_528MHZ=y
-CONFIG_RAM_512MB=y
+CONFIG_RAM_256MB=y
 # CONFIG_PMECC_AUTO_DETECT is not set
 CONFIG_PMECC_CORRECT_BITS_4=y
 CONFIG_LOAD_LINUX=y
-CONFIG_MEM_SIZE="0x10000000"
 CONFIG_DEBUG=y
 CONFIG_SCLK=y
 CONFIG_PM_EXTERNAL_DEVICES=y
diff --git a/board/sama5d3_xplained/sama5d3_xplainednf_linux_zimage_dt_defconfig b/board/sama5d3_xplained/sama5d3_xplainednf_linux_zimage_dt_defconfig
index 4b14428..cc3604b 100644
--- a/board/sama5d3_xplained/sama5d3_xplainednf_linux_zimage_dt_defconfig
+++ b/board/sama5d3_xplained/sama5d3_xplainednf_linux_zimage_dt_defconfig
@@ -1,11 +1,10 @@
 CONFIG_SAMA5D3_XPLAINED=y
 CONFIG_CPU_CLK_528MHZ=y
-CONFIG_RAM_512MB=y
+CONFIG_RAM_256MB=y
 # CONFIG_PMECC_AUTO_DETECT is not set
 CONFIG_PMECC_CORRECT_BITS_4=y
 CONFIG_LOAD_LINUX=y
 CONFIG_LINUX_ZIMAGE=y
-CONFIG_MEM_SIZE="0x10000000"
 CONFIG_DEBUG=y
 CONFIG_SCLK=y
 CONFIG_PM_EXTERNAL_DEVICES=y
diff --git a/board/sama5d3_xplained/sama5d3_xplainednf_uboot_defconfig b/board/sama5d3_xplained/sama5d3_xplainednf_uboot_defconfig
index 1aefac1..c25938b 100644
--- a/board/sama5d3_xplained/sama5d3_xplainednf_uboot_defconfig
+++ b/board/sama5d3_xplained/sama5d3_xplainednf_uboot_defconfig
@@ -1,6 +1,6 @@
 CONFIG_SAMA5D3_XPLAINED=y
 CONFIG_CPU_CLK_528MHZ=y
-CONFIG_RAM_512MB=y
+CONFIG_RAM_256MB=y
 # CONFIG_PMECC_AUTO_DETECT is not set
 CONFIG_PMECC_CORRECT_BITS_4=y
 CONFIG_DEBUG=y
diff --git a/board/sama5d3_xplained/sama5d3_xplainedsd_linux_uimage_dt_defconfig b/board/sama5d3_xplained/sama5d3_xplainedsd_linux_uimage_dt_defconfig
index 673f694..c74bfad 100644
--- a/board/sama5d3_xplained/sama5d3_xplainedsd_linux_uimage_dt_defconfig
+++ b/board/sama5d3_xplained/sama5d3_xplainedsd_linux_uimage_dt_defconfig
@@ -1,9 +1,8 @@
 CONFIG_SAMA5D3_XPLAINED=y
 CONFIG_CPU_CLK_528MHZ=y
-CONFIG_RAM_512MB=y
+CONFIG_RAM_256MB=y
 CONFIG_SDCARD=y
 CONFIG_LOAD_LINUX=y
-CONFIG_MEM_SIZE="0x10000000"
 CONFIG_DEBUG=y
 CONFIG_SCLK=y
 CONFIG_PM_EXTERNAL_DEVICES=y
diff --git a/board/sama5d3_xplained/sama5d3_xplainedsd_linux_zimage_dt_defconfig b/board/sama5d3_xplained/sama5d3_xplainedsd_linux_zimage_dt_defconfig
index d4cce90..d77cb3d 100644
--- a/board/sama5d3_xplained/sama5d3_xplainedsd_linux_zimage_dt_defconfig
+++ b/board/sama5d3_xplained/sama5d3_xplainedsd_linux_zimage_dt_defconfig
@@ -1,10 +1,9 @@
 CONFIG_SAMA5D3_XPLAINED=y
 CONFIG_CPU_CLK_528MHZ=y
-CONFIG_RAM_512MB=y
+CONFIG_RAM_256MB=y
 CONFIG_SDCARD=y
 CONFIG_LOAD_LINUX=y
 CONFIG_LINUX_ZIMAGE=y
-CONFIG_MEM_SIZE="0x10000000"
 CONFIG_DEBUG=y
 CONFIG_SCLK=y
 CONFIG_PM_EXTERNAL_DEVICES=y
diff --git a/board/sama5d3_xplained/sama5d3_xplainedsd_uboot_defconfig b/board/sama5d3_xplained/sama5d3_xplainedsd_uboot_defconfig
index 473956f..e9c62c3 100644
--- a/board/sama5d3_xplained/sama5d3_xplainedsd_uboot_defconfig
+++ b/board/sama5d3_xplained/sama5d3_xplainedsd_uboot_defconfig
@@ -1,6 +1,6 @@
 CONFIG_SAMA5D3_XPLAINED=y
 CONFIG_CPU_CLK_528MHZ=y
-CONFIG_RAM_512MB=y
+CONFIG_RAM_256MB=y
 CONFIG_SDCARD=y
 CONFIG_DEBUG=y
 CONFIG_SCLK=y
-- 
1.9.1

